packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/sdkforyourapp.md

```diff
@@ -0,0 +1,40 @@
+#### Step 1: Set up your app with prerequisites
+
+1. In your Okta Org, [create a new application integration](/docs/guides/oie-embedded-common-org-setup/java/main/#step-4-create-new-application) for your app.
+1. Ensure that you have all the [software requirements](#software-requirements).
+1. Clone the [okta-idx-java](https://github.com/okta/okta-idx-java) repository.
+1. Ensure that you have set the Okta Java SDK [dependency](#software-requirements) for Apache Maven. Use the latest release version listed [here](https://github.com/okta/okta-idx-java#release-status).
+
+#### Step 2: Import packages and add framework
+
+Both the embedded authentication with SDK and the embedded Sign-In Widget sample apps use the Spring Boot framework with the Okta Java SDK. Import the Okta API packages as well as any Sprint Boot packages you need.
```

Comment:
Typo: should be `Spring` instead of `Sprint`.

Reply:
Thanks! Will update in new branch **vn-OKTA-408182-oie-java-embedded-app-2**.